docs/DeveloperGuide.md

```diff
@@ -114,7 +114,7 @@ How the parsing works:
 * All `XYZCommandParser` classes (e.g., `AddCommandParser`, `DeleteCommandParser`, ...) inherit from the `Parser` interface so that they can be treated similarly where possible e.g, during testing.
 
 ### Model component
-**API** : [`Model.java`](https://github.com/AY2122S2-CS2103T-T17-4/tp/tree/master/src/main/java/seedu/address/model/Model.java)
+**API** : [`Model.java`](images/ModelClassDiagram.png)
```

Comment:
BetterModelClassDiagram at line 131 needs to be updated with `PersonAttribute` as well 😃